src/DataLoader.Uno.WinUI/DataLoaderView.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.UI.Xaml;
```

Comment:
Could we share code for `DataLoaderView `and `DataLoaderViewController` using global usings and base classes?